modern/src/store/devices.js

```diff
@@ -20,6 +21,14 @@ const { reducer, actions } = createSlice({
     remove(state, action) {
       delete state.items[action.payload];
     },
+    filterByName(state, action) {
```

Comment:
Can filtering be done in the component instead of global?

Reply:
I think that could be done that way, but it will require to pass params down to components and to store a filtered list of devices in the component state or to use Context API.

Which one do you think is best?

Reply:
Ok, I'll work on it.